modules/_labs/select/react/stories/stories.tsx

```diff
@@ -296,29 +295,38 @@ storiesOf('Labs|Select/React/Visual Testing', module)
         ]}
         columnProps={permutateProps(
           {
+            'aria-selected': [{label: '', value: undefined}, {label: 'Selected', value: 'true'}],
             className: [
               {label: 'Default', value: ''},
               {label: 'Hover', value: 'hover'},
               {label: 'Active', value: 'active'},
               {label: 'Active Hover', value: 'active hover'},
             ],
             disabled: [{label: '', value: false}, {label: 'Disabled', value: true}],
+            focused: [{label: '', value: false}, {label: 'Assistive-focus', value: true}],
           },
           props => {
-            if (props.disabled && !['', 'hover'].includes(props.className)) {
+            if (
+              props.disabled &&
+              (props['aria-selected'] === 'true' ||
+                !['', 'hover'].includes(props.className) ||
+                props.focused)
+            ) {
               return false;
             }
             return true;
```

Comment:
The downside of using `permuateProps` is a very difficult to understand prop filtering. I think this means to not show column (unclear why column and not row) if the the prop merged-matrix has both `disabled` AND either `aria-selected` or default/hover classNames or focused.

Reply:
That's correct, although FWIW I'm no longer using `permutateProps` in the latest version of the states table. Given the smaller number of combinations for the columns and rows in the new tables, I didn't think it was necessary to use `permutateProps`.